src/app/patient-dashboard/patient-dashboard.module.ts

```diff
@@ -33,7 +32,7 @@ import { PatientIdentifierComponent } from './patient-identifier/patient-identif
 import { ProgramService } from './programs/program.service';
 import { ProgramsComponent } from './programs/programs.component';
 import { AppFeatureAnalytics } from '../shared/app-analytics/app-feature-analytics.service';
-import { PatientSearchService } from './patient-search/patient-search.service';
+import { PatientSearchService } from '../patient-search/patient-search.service';
```

Comment:
This should not be here as it is already being provided by the patient search module